src/oc_erchef/apps/chef_index/src/chef_index_queue.erl

```diff
@@ -103,10 +107,29 @@ package_for_delete(Type, ID, DatabaseName, SolrUrl) ->
   InnerEnvelope = inner_envelope(Type, ID, DatabaseName, {[]}, SolrUrl),
   {[{action, delete}, {payload, InnerEnvelope}]}.
 
+
+create_management_pool(Username, Password, Config) ->
+    chef_wm_rabbitmq_management:create_pool(?POOLNAME, add_basic_auth(Username, Password, Config)).
+
+-spec ping(binary()) -> pong | pang.
+ping(VHost) ->
+    % TODO: chef_wm_rabbitmq_management needs to be moved to a shared app
```

Comment:
[nit] The TODO is fine but we are trying to use `TODO(ssd) 2017-08-02: ` so we know how long they've been there and who added them without crawling through git history.